start/src/main/java/org/apache/accumulo/start/Main.java

```diff
@@ -212,6 +212,7 @@ public static void printUsage() {
       printCommand(executableMap.remove(cmd));
     }
     System.out.println("  <main class> args              Runs Java <main class> located on Accumulo classpath");
+    System.out.println("                                 Add jars to the classpath using '-add jar1:jar2'");
```

Comment:
I don't think the java code should be documenting the wrapper script, which may or may not be used.
I also think this feature can be removed, since it's useless.